### DISCUSSION TITLE
is anyone else extraction freezing?

### DISCUSSION THREAD

**Discussion by Jayspency** [KadeArchive/Kade-Engine#2142] [Q&A]
idk if anyone else is having this issue but when iI extract my game, it freezes at Evil_Mall or Bfgameover.(around 2% or 12%). I have not had any issue with version 1.7 in the past. It just started happening today

**Comment by BoloVEVO** [comment]
Idk but 1.7 is very buggy and I don't feel comfortable playing in that version. Use instead 1.6.2

**Reply by Jayspency** [reply]
ok also the stage code is alot different it doesnt even show the actual code for it
